glue/viewers/image/qt/slice_widget.py

```diff
@@ -89,11 +89,10 @@ def sync_sliders_from_state(self, *args):
                 # check the type of data.coords here since we want to treat
                 # subclasses differently.
                 if getattr(self.data, 'coords') is not None and type(self.data.coords) != LegacyCoordinates:
-                    world_axis_index = self.data.ndim - 1 - i
                     world = world_axis(self.data.coords, self.data,
-                                       pixel_axis=world_axis_index,
-                                       world_axis=world_axis_index)
-                    world_unit = self.data.coords.world_axis_units[world_axis_index]
+                                       pixel_axis=self.data.ndim - i - 1,
+                                       world_axis=self.data.ndim - i - 1)
+                    world_unit = self.data.coords.world_axis_units[self.data.ndim - i - 1]
```

Comment:
Can you revert these changes? This was just fixed in https://github.com/glue-viz/glue/pull/2159

Reply:
My apologies, there was some problem with my IDE... I have already rebased but the changes were not picked up. Let me check again to see if I could fix it.

Reply:
Hey @astrofrog I tried but it seems very difficult to untangle the mess I have created since the commit I have made for this change is deep within the PR... Is there any workarounds you would suggest me do to fix this? Any suggestion would be much appreciated.

Reply:
No worries, change reverted 👍

Reply:
Sorry the code looked eerily like my own, as I have submitted a very similar commit before the patch PR #2159 was opened, even with the same variable name, so I was confused 😐 Now the issue has been fixed.